be1-go/message/messagedata/mod.go

```diff
@@ -69,6 +69,12 @@ type MessageData interface {
 	NewEmpty() MessageData
 }
 
+// VerifiableMessageData defines a MessageData that offers message verification
+type VerifiableMessageData interface {
```

Comment:
The name should be only `Verifiable`, because it is defined in a `messagedata` package. Hence when you use the interface you end up with `messagedata.Verifiable` instead of `messagedata.VerifiableMessageData`.